src/test/java/K_pages/LoginPage.java

```diff
@@ -0,0 +1,43 @@
+package K_pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+public class LoginPage extends BasePage {
+    WebDriver driver;
+    WebDriverWait wait;
+    //    locators
+    By submitButtonLocator = By.cssSelector("[type='submit']");
+    By emailFieldLocator = By.cssSelector("[type='email']");
+    By passwordFieldLocator = By.cssSelector("[type='password']");
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```